src/main/java/edu/cornell/kfs/fp/businessobject/ScheduledAccountingLine.java

```diff
@@ -0,0 +1,28 @@
+package edu.cornell.kfs.fp.businessobject;
+
+import java.sql.Date;
+
+import org.kuali.kfs.sys.businessobject.AccountingLine;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+public interface ScheduledAccountingLine extends AccountingLine {
+	public abstract String getPartialTransactionCount();
+    
+    public abstract void setPartialTransactionCount(String partialTransactionCount);
```

Comment:
The "public" and "abstract" modifiers are redundant for interface methods, so I'd probably remove them.